tests/iface_loopback_action/iface_loopback_action_helper.py

```diff
@@ -169,16 +169,23 @@ def remove_orig_dut_port_config(duthost, orig_ports_configuration):
     :param duthost: DUT host object
     :param orig_ports_configuration: original ports configuration parameters
     """
+    remove_acl_tables(duthost)
     for _, port_dict in list(orig_ports_configuration.items()):
         port = port_dict['port']
         if port_dict['vlan']:
             remove_dut_vlan_member(duthost, port, port_dict['vlan'])
         elif port_dict['portchannel']:
             remove_dut_portchannel_member(duthost, port, port_dict['portchannel'])
+            output = duthost.shell('show interface portchannel | grep "{} "'.format(port_dict['portchannel']))
```

Comment:
are they needed to be restored? if not, why do they exist at the first place?

Reply:
Hi Roy, the configuration will be restored by the autouse fixture:
https://github.com/sonic-net/sonic-mgmt/blob/1f349d57dc3e6ac883c105a8747cd275f2bc1590/tests/iface_loopback_action/conftest.py#L211-L220